src/Diagnostics/diagnostic_vars.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    Diagnostics variable template
+
+Container for diagnostic variables of interest. Useful also for 
+post-processing.
+
+"""
+
+using CLIMA.VariableTemplates
+
+function vars_diagnostic(FT)
+  @vars begin
+    # vertical coordinate
+    z::FT
+    # state and functions of state
+    u::FT
+    v::FT
+    w::FT
+    q_tot::FT
+    e_tot::FT
+    q_liq::FT
+    θ::FT
+    θ_liq::FT
+    θ_v::FT
+    e_int::FT
+    # vertical fluxes
+    w′ρ′::FT
+    w′u′::FT
+    w′v′::FT
+    w′q_tot′::FT
+    w′q_liq′::FT
+    w′q_vap′::FT
+    w′θ′::FT
+    w′θ_v′::FT
+    w′θ_liq′::FT
+    # variances
+    u′u′::FT
+    v′v′::FT
+    w′w′::FT
+    # skewness
+    w′w′w′::FT
```

Comment:
Although we use symbols within the main clima code, it would be better to not use symbols for the variables that are written to file. 
Not every user may be able to use symbols when calling @kpamnany 's new graph_diagnostics.jl from the command line
(Notice the graph_diagnostics.jl is passed the list of variables to be plotted:
`>> julia graph_diagnostics.jl <diagnostic_file.jld2> <diagnostic_name>` where  
 <diagnostic> is a list of the variables to be plotted.

In atmospheric software packages we usually find names such as:
POTTEMP, VIRT_POTTEMP, LIQ_POTTEMP, etc.
Bottom line, I would totally avoid greek symbols to name the diagnostic variables.

Reply:
Added `altvars_diagnostic`, as a mirror of `vars_diagnostic`, with some of the changes suggested here. If you give me a full list of what you'd like changed, I'll make the changes and then update `graph_diagnostic.jl` to use `altvars_diagnostic` instead, which will allow you to use non-unicode names in post-processing while still allowing the CLIMA code to use unicode.

Reply:
Not to make this go back and forth but, I think a better solution here would be to write mapping functions for each symbol/expression, and calls to print/show can be intercepted by the mapping funcs, instead of having two structs of the same thing.

Reply:
Something like:
```
struct VerboseName{T} end
VerboseName(T) = VerboseName{T}()

nice_name(::VerboseName{:θ})        = :theta
nice_name(::VerboseName{:e_int})    = :e_int

struct Foo{FT}
  θ::FT
  e_int::FT
end

nice_names(f::Foo) = nice_name.(VerboseName.(fieldnames(typeof(f))))

f = Foo(1.0,2.0)

@show nice_names(f)
```

Reply:
Agreed. In the diagnostics issue, I outlined what we’ll need in the end: a table that defines short and verbose names for the output fields, and units, to be able to generate self-documenting output. May not need to be now, but that’s where we’ll need to go eventually.

Reply:
Having primes etc. in the code is fine for now though. I wouldn’t go through the trouble to create more conventional variable names in the code right now.

Reply:
I'll just switch to the conventional variable names now, since we want to do that anyway.

Reply:
> I'll just switch to the conventional variable names now, since we want to do that anyway.

If by conventional you mean the long names @smarras79 suggested, my take is that is not needed right now. There are no general conventions for the names, and your primed notation had the advantage of clarity for now. Most importantly, it think it would be good to get this merged, and then we can refine.

Reply:
Simone has been working with a modified version of this code with plain text names due to difficulties with specifying the Unicode names we're using. I've just switched to his version; not a difficult thing. I agree on getting this merged ASAP!